bin/pulsar

```diff
@@ -88,7 +90,8 @@ Environment variables:
    PULSAR_BROKER_CONF            Configuration file for broker (default: $DEFAULT_BROKER_CONF)
    PULSAR_BOOKKEEPER_CONF        Configuration file for bookie (default: $DEFAULT_BOOKKEEPER_CONF)
    PULSAR_ZK_CONF                Configuration file for zookeeper (default: $DEFAULT_ZK_CONF)
-   PULSAR_GLOBAL_ZK_CONF         Configuration file for global zookeeper (default: $DEFAULT_GLOBAL_ZK_CONF)
+   PULSAR_GLOBAL_ZK_CONF         Deprecated. Configuration file for global configuration zookeeper (default: $DEFAULT_GLOBAL_ZK_CONF)
```

Comment:
`Configuration file for global zookeeper (default: $DEFAULT_GLOBAL_ZK_CONF) (Deprecated. Replace with "PULSAR_CONFIGURATION_STORE_CONF")`